internal/service/resources/db/models/converters.go

```diff
@@ -0,0 +1,82 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/openshift-kni/oran-o2ims/internal/service/resources/api/generated"
+)
+
+// DeploymentManagerToModel converts a DB tuple to an API Model
+func DeploymentManagerToModel(record *DeploymentManager) (generated.DeploymentManager, error) {
```

Comment:
I would delete the error here and in all the XToY functions. It adds unneeded code (checking error that is always nil) in server.go

Reply:
I can remove them for now, but i expect that some of them will need the error to convert the enums and handled unexpected value.  ...for now I agree this is unnecessary.  ...i'll deal with it in the next PR.

Reply:
this will be addressed in a future PR (discussed internally)